lib/msf/base/sessions/mettle_config.rb

```diff
@@ -9,6 +9,20 @@ module MettleConfig
 
       include Msf::Payload::TransportConfig
 
+      def initialize(info = {})
+        super
+
+        register_advanced_options(
+          [
+            OptBool.new(
+              'MeterpreterTryToFork',
+              'Fork a new process if the functionality is available',
+              default: false
```

Comment:
In the future we might want to investigate setting this to `true`, but that seems like a separate effort :+1:

https://github.com/rapid7/metasploit-framework/blob/364cb1336250ac30c5338350e4e42a6b4076b9c8/lib/msf/core/payload/python/meterpreter_loader.rb#L31-L35